### PR TITLE
Decapitalized log.Debug messages

### PR DESCRIPTION
**Pull request by RichardLitt**
According to golang standards, these should not be capitalized nor having a trailing period, AFAIK.

License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

**Comment by Kubuxu** [comment]
I don't know golang standards that well, but SGTM.

**Comment by RichardLitt** [comment]
@Kubuxu Me neither. But @whyrusleeping consistently stated that errors should be lowercase - I figure this should be similar.

File: fuse/ipns/ipns_unix.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func NewFileSystem(ipfs *core.IpfsNode, sk ci.PrivKey, ipfspath, ipnspath string
 
 // Root constructs the Root of the filesystem, a Root object.
 func (f *FileSystem) Root() (fs.Node, error) {
-	log.Debug("Filesystem, get root")
+	log.Debug("gilesystem, get root")
```
Comment (deltab) [inline fuse/ipns/ipns_unix.go]: Typo ­– should be `filesystem`.

(Your editor may have a 'convert to lowercase' command that avoids this.)

Comment (RichardLitt) [inline fuse/ipns/ipns_unix.go]: Good catch, and good call. Thanks.

**Comment by whyrusleeping** [comment]
this LGTM

**Pull request merged**